src/pages/About/index.tsx

```diff
@@ -27,7 +28,7 @@ const PageBackground = styled.div<{ isDarkMode: boolean }>`
   position: absolute;
   width: 100%;
   height: 100vh;
-  top: 0;
+  top: -72px;
```

Comment:
I think you can replace this with a navHeight from the theme?

Reply:
yes